pytition/pytition/wsgi.py

```diff
@@ -14,3 +14,33 @@
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "pytition.settings")
 
 application = get_wsgi_application()
+
+if os.environ.get('EMAIL_BACKEND') == 'mailer':
+    from django.conf import settings
+    try:
+        import uwsgidecorators
+        from django.core.management import call_command
+
+        @uwsgidecorators.timer(settings.UWSGI_WAIT_FOR_MAIL_SEND_IN_S)
+        def send_queued_mail(num):
+            """Send queued mail every 10 seconds"""
+            call_command('send_mail')
+
+        @uwsgidecorators.timer(settings.UWSGI_WAIT_FOR_RETRY_IN_S)
+        def retry_deferred(num):
+            """Retry deferred emails"""
+            call_command('retry_deferred')
+
+        @uwsgidecorators.timer(settings.UWSGI_WAIT_FOR_PURGE_IN_S)
+        def purge_mail(num):
+            """purge 2 days old mails"""
+            call_command('purge_mail_log', settings.UWSGI_NB_DAYS_TO_KEEP)
+
+    except ImportError:
+        print("""uwsgidecorators not found. Cron are disabled,
```

Comment:
I guess this ImportError will be triggered if not using uwsgi.
What happens if you still want to use the mailer, without uwsgi and with crontab jobs like written by the prints?

Reply:
without uwsgi it should work in the same way if you add the cron jobs.

maybe that it not clear enough, do you see a better way to explain this somewhere?